packages/playwright-test/src/plugins/index.ts

```diff
@@ -0,0 +1,17 @@
+/**
+ * Copyright (c) Microsoft Corporation.
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ * http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+export { webServer } from './webServerPlugin';
+export { createPlugin as vite } from './vitePlugin';
```

Comment:
Ignore the import/export related code as that will all change before shipping. (i.e. do we ship as individual packages `@playwright/test-plugin-vite`, `@playwright/test-plugin-webserver`, …? We will likely want to do this long term to not bloat the @playwright/test bundle.)